.gitignore

```
@@ -364,3 +365,9 @@ Cargo.lock
 
 # MSVC Windows builds of rustc generate these, which store debugging information
 *.pdb
+
+# Ignore the globalPackagesFolderTest
+**/globalPackagesFolderTest/
```

```suggestion
globalPackagesFolderTest/
```

nit: can be simplified per the docs:

> For example, a pattern doc/frotz/ matches doc/frotz directory, but not a/doc/frotz directory; however frotz/ matches frotz and a/frotz that is a directory (all paths are relative from the .gitignore file).

Source: https://git-scm.com/docs/gitignore